Validation: Added Missing Descriptions (#241)
MicrosoftDocs/bingmaps-docs: BingMaps/spatial-data-services/data-source-management-api/load-data-source-dataflow/load-data-source-dataflow-sample-code-csharp.md

---
title: "Load Data Source Dataflow Sample Code (C#) | Microsoft Docs"

> description: "Provides Load Data Source Dataflow sample C# code for classes that upload spatial data and check status of dataflow jobs and support classes."

Reviewer: ```suggestion
description: C# sample code demonstrating how to upload entity data to a data source using the Load Data Source Dataflow
```
